models/survey.py

```diff
@@ -1,52 +1,32 @@
+from sqlalchemy import Column, ForeignKey, Integer, String, Date
+# from sqlalchemy.ext.declarative import declarative_base
 import datetime
 
-from app import app, db
-from flask import flash, render_template, request, redirect
-from wtforms import Form, IntegerField, StringField
+from config.helper import Base
 
-class Survey:
+class Survey(Base):
     __tablename__ = "surveys"
- 
-    id = int()
-    description = str()
-    start_date = datetime.date
-    end_date = datetime.date
+
+    # Mappers
+    id_ = Column(Integer, primary_key=True)
+    description = Column(String, nullable=False)
+    start_date = Column(Date)
+    end_date = Column(Date)
 
     def __init__(self, id_, description_):
-        self.id = id_
+        self.id_ = id_
         self.description = description_
 
     def __repr__(self):
-        return "<Question: {}>".format(self.name)
-
-class surveyForm(Form):
-    id = IntegerField("ID")
-    description = StringField("Description")
-
-@app.route('/addNewSurvey', methods=['GET', 'POST'])
-def addNewSurvey():
-    form = surveyForm(request.form)
-
-    if request.method == 'POST' and form.validate():
-        new_survey = db.prepare("INSERT INTO survey VALUES ($1, $2)")
-        with db.xact():
-        	new_survey(form.id.data, form.description.data)
-        
-        flash("Survey " + str(form.id.data) + " " +  \
-            form.description.data + " saved successfully!")
-        return redirect('/')
-
-    return render_template('new_survey.html', form=form)
-
-@app.route('/printSurveys', methods=['GET', 'POST'])
-def printSurveys():
-    surveys = []
-    form = surveyForm(request.form)
-
-    all_the_surveys = db.prepare("SELECT * FROM survey")
-    
-    with db.xact():
-      for survey in all_the_surveys():
-        surveys.append(survey[1])
-    
-    return render_template('surveys.html', surveys=surveys)
\ No newline at end of file
+        return "<Question: {}>".format(self.description)
```

Comment:
Shouldn't it return a Survey obj?

Reply:
Apparently it can be a string as well, so I went with Peter's original definition. Depends on what is needed. 

From the documentation: **repr(object)**: Return a string containing a printable representation of an object. (...) For many types, this function makes an attempt to return a string that would yield an object with the same value when passed to eval(), otherwise the representation is a string enclosed in angle brackets that contains the name of the type of the object together with additional information often including the name and address of the object. 

And **Peter**, I'm obviously not suggesting that your routes be destroyed, only edited to be compatible with ORM and then placed wherever routes will be, so that's why they're temporarily missing.

Reply:
@InkaSimola Actually my point was the type that's defined here. This line is in survey's code, but returns a question. I think this was a typo in Peter's original code. Could you fix it in this branch?